src/main/kotlin/graphics/scenery/volumes/Volume.kt

```diff
@@ -940,7 +986,12 @@ open class Volume(
          *
          * Returns the new volume.
          */
-        @JvmStatic fun fromPathRaw(file: Path, hub: Hub): BufferedVolume {
+        @JvmStatic @JvmOverloads
+        fun <T: RealType<T>> fromPathRaw(
+            file: Path,
+            hub: Hub,
+            type: T
```

Comment:
I'd add a default argument here, where `type: T = UnsignedByteType()` 👍 
That'd also make the changes in the example unnecessary.

Reply:
I'm not sure if it is possible to provide a default argument for a template parameter? The code does not compile with the exact change you proposed @skalarproduktraum